doc: Document claiming devices using claim auth code

nejraselimovic opened the pull request:
#### Summary
Closes #1079 

#### Notes for Reviewers
Is this good place to locate these instructions? I find it a bit weird that it's separated from QR code-related docs and adding devices, but I somehow don't see these in Adding Devices. Also I'm curious, why does `join-server-enabled` option have to be set to false?

#### Checklist
- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

KrishnaIyer reviewed (commented):
Claiming a device using the QR Code from the Consle is already documented in the [main flow](https://github.com/TheThingsIndustries/lorawan-stack-docs/blob/master/doc/content/devices/adding-devices/_index.md). 
I think claiming using the CLI should go into the [Bulk section](https://github.com/TheThingsIndustries/lorawan-stack-docs/tree/master/doc/content/devices/adding-devices/adding-devices-in-bulk) since you can automate claiming for multiple devices.
I would remove this page entirely and add a alias in the [main flow](https://github.com/TheThingsIndustries/lorawan-stack-docs/blob/master/doc/content/devices/adding-devices/_index.md).

nejraselimovic commented:
> Claiming a device using the QR Code from the Consle is already documented in the [main flow](https://github.com/TheThingsIndustries/lorawan-stack-docs/blob/master/doc/content/devices/adding-devices/_index.md). I think claiming using the CLI should go into the [Bulk section](https://github.com/TheThingsIndustries/lorawan-stack-docs/tree/master/doc/content/devices/adding-devices/adding-devices-in-bulk) since you can automate claiming for multiple devices. I would remove this page entirely and add a alias in the [main flow](https://github.com/TheThingsIndustries/lorawan-stack-docs/blob/master/doc/content/devices/adding-devices/_index.md).

I moved this to adding devices in bulk section and added info on how to claim both single and multiple devices, so all options are documented. I also think we should keep the **Claiming devices** doc as it has interesting info about the general concept of claiming.

KrishnaIyer commented on doc/content/devices/adding-devices/adding-devices-in-bulk/_index.md:
```diff
@@ -87,3 +87,79 @@ See the following video from [The Things Network youtube channel](https://youtu.
 {{< youtube "ouz-VuiosU4" >}}
 </details>
+
+## Claim devices using Claim Authentication Code
+
+Devices can be added in bulk together with their ownership information, through a process of device claiming using their claim authentication codes.
+
+Claiming devices can be done using {{% tts %}} Console or the CLI, for single or multiple devices.
+
+{{< tabs/container "Console" "CLI" >}}
+
+{{< tabs/tab "Console" >}}
+
+To claim a single device, first navigate to your application in {{% tts %}} Console and click the **+ Register end device** button.
+
+Select the **Enter end device specifics manually**, then provide **Frequency plan**, **LoRaWAN version** and **Regional Parameters version**.
+
+In the **Provisioning information** section, fill in the **JoinEUI** of The Things Join Server and **Confirm**. 
+
+Then, enter the **DevEUI**, **Claim authentication code** and **End device ID** of a device that you wish to claim.
```
But what is the difference between this and the `## Onboarding devices using QR codes` in the `adding-devices`page?

KrishnaIyer replied:
Oh I see. Sorry, you're right then.
